service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/RegistryManager.java

```diff
@@ -419,13 +419,15 @@ public Device updateDevice(Device device) throws IOException, IotHubException
 
     /**
      * Update device with forceUpdate input parameter
+     * @deprecated Please use {@link #updateDevice(Device)}
```

Comment:
I'd add an explanation here. Something like "the forceUpdate argument does nothing so this method will always behave the same as {link updateDevice(Device)}"

Reply:
yeah sounds good, will add that